test/CMakeLists.txt

```diff
@@ -32,6 +32,23 @@ else()
     message(STATUS "oneDPL: -fno-fast-math is not supported by current compiler")
 endif()
 
+# usage of Kernel names in tests
+set(TestExplicitKernelNameVariants AUTO ALWAYS)
+set(ONEDPL_EXPLICIT_KERNEL_NAMES AUTO CACHE STRING "Use Kernel names in tests")
```

Comment:
Please see the example at https://stackoverflow.com/questions/39670879/cmake-variable-with-limited-valid-values